functorch/_src/aot_autograd.py

```diff
@@ -1742,6 +1742,8 @@ def aot_module_simplified(
     """
     #########################################################
 
+    torch._dynamo.utils.assert_no_fake_params_or_buffers(mod)
```

Comment:
A little comment explaining why it's invalid to pass fakeified modules to aot_module_simplified would be nice!

Reply:
Done